modules/stochastic_tools/src/userobjects/StateSimRunner.C

```diff
@@ -0,0 +1,68 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "StateSimRunner.h"
+#include <string>
+
+template <>
+InputParameters
+validParams<StateSimRunner>()
+{
+  InputParameters params = validParams<GeneralUserObject>();
+  params.addRequiredParam<std::string>("model_path", "the model location for the state simulation");
+  return params;
+}
+
+StateSimRunner::StateSimRunner(const InputParameters & parameters)
+  : GeneralUserObject(parameters),
+    _model_path("model_path"),
```

Comment:
This is fine, we need to find an open source JSON reader with a LGPL 2.1 compatible license. BTW - You'll want to go to your PR on the web to respond to comments. Responding through email doesn't seem to put them in the right place.